truss/test_data/truss_container_fs/app/model_wrapper.py

```diff
@@ -107,27 +106,27 @@ def try_load(self):
 
     def preprocess(
         self,
-        payload: Union[Dict, CloudEvent, ModelInferRequest],
+        payload: Any,
```

Comment:
Isn't this always at least a Dict?

Reply:
No. We dropped that requirement from Philips last big PR that dropped input validation